sources/Interop/Windows/um/dxcapi/DXC.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from DirectXShaderCompiler, include/dxc/dxcapi.h, at commit d356a8bc546da38612cac13d1e643de17a0ba38f
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class DXC
+    {
+        private const string LibraryPath = "DXC";
+
+        [DllImport(LibraryPath, CallingConvention = CallingConvention.StdCall, EntryPoint = "DxcCreateInstance", ExactSpelling = true)]
+        [return: NativeTypeName("HRESULT")]
+        public static extern int DxcCreateInstance([NativeTypeName("const IID &")] Guid* rclsid, [NativeTypeName("const IID &")] Guid* riid, [NativeTypeName("LPVOID *")] void** ppv);
+
+        [DllImport(LibraryPath, CallingConvention = CallingConvention.StdCall, EntryPoint = "DxcCreateInstance2", ExactSpelling = true)]
+        [return: NativeTypeName("HRESULT")]
+        public static extern int DxcCreateInstance2([NativeTypeName("IMalloc *")] IMalloc* pMalloc, [NativeTypeName("const IID &")] Guid* rclsid, [NativeTypeName("const IID &")] Guid* riid, [NativeTypeName("LPVOID *")] void** ppv);
+
+        // [NativeTypeName("const UINT32")]
+        // public static extern uint DxcValidatorFlags_Default;
```

Comment:
All of these should be under an:
```
--exclude
DxcValidatorFlags_Default
DxcValidatorFlags_InPlaceEdit
...
```

They should then be added to a `Windows.Manual.cs` with the appropriate value.
For the GUID, they should be converted to the non-string constructor version.